kfs-prje/src/main/resources/edu/arizona/kfs/module/prje/spring-prje.xml

```diff
@@ -27,10 +27,186 @@
+        <property name="initialPageNumber" value="1" />
+        <property name="pageLabel" value="PAGE:" />
+        <property name="newLineCharacter" value="&#xA;" />
+        <property name="dateTimeService" ref="dateTimeService" />
+
+        <property name="classToBusinessObjectReportHelperBeanNames">
+            <map>
+                <entry key="edu.arizona.kfs.module.prje.businessobject.PRJESet"
+                       value="prjeSetReportHelper" />
+                <entry key="edu.arizona.kfs.module.prje.businessobject.PRJEType"
+                       value="prjeTypeReportHelper" />
+                <entry key="edu.arizona.kfs.module.prje.businessobject.PRJEAuditItem"
+                       value="prjeAuditItemReportHelper" />
+            </map>
+        </property>
+    </bean>
+
+    <bean id="prjeReports" parent="prjeReports-parent"/>
+    <bean id="prjeReports-parent" abstract="true" 
+          class="edu.arizona.kfs.module.prje.service.impl.PRJEReportsServiceImpl">
+       <property name="prjeDataAccess" ref="prjeDataAccess" />
+       <property name="businessObjectService" ref="businessObjectService" />
+       <property name="parameterService" ref="parameterService" />
+       <property name="objectCodeService" ref="objectCodeService" />
+       <property name="dateTimeService" ref="dateTimeService" />
+       <property name="universityDateService" ref="universityDateService" />
+       <property name="batchFileDirectoryName" value="${staging.directory}/gl/originEntry"/>
+       <property name="ledgerReportWriterService" ref="prjePostingLedgerReportWriterService" />
+       <property name="auditReportWriterService" ref="prjeAuditReportWriterService" />
+    </bean>
+
+	<bean id="prjeService" parent="prjeService-parent"/>
+	<bean id="prjeService-parent" abstract="true"
+	      class="edu.arizona.kfs.module.prje.service.impl.PRJEServiceImpl">
+	   <property name="prjeDataAccess" ref="prjeDataAccess" />
+       <property name="businessObjectService" ref="businessObjectService" />
+       <property name="parameterService" ref="parameterService" />
+       <property name="objectCodeService" ref="objectCodeService" />
+       <property name="dateTimeService" ref="dateTimeService" />
+       <property name="universityDateService" ref="universityDateService" />
+       <property name="batchFileDirectoryName" value="${staging.directory}/gl/originEntry"/>
+       <property name="propertyFileDirectoryName" value="${staging.directory}/prje"/>
+       <property name="prjeReports" ref="prjeReports" />
+	</bean>
+
+	<bean id="prjeJob" parent="unscheduledJobDescriptor">
+		<property name="steps">
+			<list>
+				<ref bean="prjeTransactionStep" />
+			</list>
+		</property>
+	</bean>
+
+	<bean id="prjeTransactionStep" parent="step"
+	      class="edu.arizona.kfs.module.prje.batch.PRJETransactionStep">
+		<property name="prjeService" ref="prjeService" />
+	</bean>
+
+	<bean id="prjeJobTrigger" parent="cronTrigger">
```

Comment:
I don't believe this trigger is needed as the prjeJob (as all our jobs are) is unscheduled.